src/Packagist/WebBundle/Resources/views/Package/viewPackage.html.twig

```diff
@@ -212,9 +316,6 @@
                 <h1>README</h1>
                 {{ package.readme|raw }}
             </div>
-            <div class="hidden readme-expander">
```

Comment:
Why was the expanded removed? Is the readme no longer partially hidden if it is very  large?